This patch fixes a spelling error in privsep.c

--- ipsec-tools-0.7.3.orig/src/racoon/privsep.c	2010-02-23 09:52:42.000000000 +0100
+++ ipsec-tools-0.7.3/src/racoon/privsep.c	2010-02-23 09:52:59.000000000 +0100
@@ -1089,7 +1089,7 @@
 }
 
 /*
- * Filter unsafe environement variables
+ * Filter unsafe environment variables
  */
 static int
 unsafe_env(envp)
@@ -1110,7 +1110,7 @@
 	return 0;
 found:
 	plog(LLV_ERROR, LOCATION, NULL, 
-	    "privsep_script_exec: unsafe environement variable\n");
+	    "privsep_script_exec: unsafe environment variable\n");
 	return -1;
 }
 
